src/legacy/state/orders/utils.ts

```diff
@@ -378,3 +378,16 @@ export function partialOrderUpdate({ chainId, order }: UpdateOrderParams, dispat
   }
   dispatch(updateOrder(params))
 }
+
+// Checks if the order is expired or not
+export const isExpired = (order: Order | undefined) => {
```

Comment:
Nitpick:
It's not a good idea to add code to the `legacy` directory.

The name of the function is too generic, can we name it as `isExpiredOrder`?

Reply:
Updated the name.
Agreed, but the where this change happens is also in legacy, so I think it makes sense

Reply:
In Nenad's defence, this is the same pattern as other stuff in this file.

Although I agree the name is not very explicit even more so because in the same file there's a `isOrderExpired` https://github.com/cowprotocol/cowswap/blob/628c62596d65e0761ccf70677a55bec9a0a36411/src/legacy/state/orders/utils.ts#L60-L59